core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1569,6 +1569,15 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_FLUSH_BATCH_SIZE =
+      new Builder(Name.MASTER_EMBEDDED_JOURNAL_FLUSH_BATCH_SIZE)
+          .setDefaultValue("1MB")
+          .setDescription("Embedded journal flush journal entries in batches to improve flush performance. "
```

Comment:
I cannot tell what the difference is between this parameter and `alluxio.master.embedded.journal.flush.size.max`. They look like very similar parameters. Could you add the differences and relationship (if there is any) between the two?

Reply:
That's true. Actually, it should be related to the journal entry size max, so i change to directly induct from the journal entry size max.

Journal entry size max defines the hard limit of a single journal entry allowed in ratis.
We need to define the single journal size max to be flushed to the ratis, could be set to 1/2 or 1/3 of the hard limit provided.